src/includes/helpers/class-alma-constants-helper.php

```diff
@@ -43,9 +44,12 @@ class Alma_Constants_Helper {
 	const PAYMENT_METHOD                 = 'payment_method';
 	const DEFAULT_CHECK_VARIATIONS_EVENT = 'check_variations';
 	const AMOUNT_PLAN_KEY_REGEX          = '#^(min|max)_amount_general_[0-9]+_[0-9]+_[0-9]+$#';
+	const SORT_PLAN_KEY_REGEX            = '/^(general|pos)_([0-9]+)_([0-9]+)_([0-9]+)$/';
```

Comment:
you can use (\d{1,2}) instead of ([0-9]+).
([0-9]+) match with : "general_122_1321_1123"

Reply:
nope, wordpress standards